parking_permits_app/resolvers.py

```diff
@@ -154,6 +140,24 @@ def resolve_update_parking_permit(obj, info, customer_id, permit_id, input):
     return {"success": True, "permit": resolve_prices_and_low_emission(permit)}
 
 
+def get_customer_permits(customer_id):
+    try:
+        permits = ParkingPermit.objects.filter(customer__pk=customer_id).order_by(
+            "start_time"
+        )
+        payload = {
+            "success": True,
+            "permits": map(resolve_prices_and_low_emission, permits),
```

Comment:
Usually, list comprehension is preferred over `map` in the python world

`"permits": [resolve_prices_and_low_emission(permit) for permit in permits],`

Reply:
Done